pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -0,0 +1,47 @@
+[id='pantheon-yaml-file_{context}']
+
+= The Pantheon configuration file
+
+You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that Pantheon can use to import content. This yaml file provides necessary information that Pantheon uses to interact with files in the repository.
+
+The configuration file enables the following:
+
+* Importing content from a Git repository into Pantheon
+* Uploading modules into Pantheon using the Uploader tool
+
+The configuration file defines the following:
+
+`server`:: The URL of the Pantheon server.
+`repository`:: A unique ID that Pantheon uses to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
+`modules`:: Patterns for modules that will show up in the search results on Pantheon.
+`resources`:: Image files, legal information or other content that is not a complete module or assembly.
+
+You need to define the values for `server` and `repository`.
+
+.Example configuration file
+
+// [options="nowrap" subs="normal"]
+----
+# Config file for Pantheon
+## server: Pantheon server URL
+## repository: a unique name, which is visible in the user facing URL
+
+## Note: Due to yaml syntax, any filepaths that start with a wildcard must be surrounded in quotes like so:
+# modules:
+#  - '*.adoc'
+
+server: {server_url}
+# Example
+#server: http://pantheon.redhat.com[]
+repository: repo_id
+
+modules:
+ - master.adoc
+ - modules/*.adoc
+
+resources:
+ - shared/legal.adoc
+ - shared/foreword.adoc
+ - resources/*
+''')
```

Comment:
I think including the `''')` here is a mistake.

Reply:
@benradey I copied this directly from the sample file. So if this is a bug, then I guess you have to fix it in the sample file too.

Reply:
Is the request here to delete line 46?

Reply:
Yes, it looks like this was copy/pasted from the python source, and line 46 with the triple-quote is the signal to python that that's where the content ends.

Reply:
I removed  `''')`